### ISSUE TITLE
Robotics page - altinak missing link

### ISSUE THREAD

**Issue by PeterGhimself** [spaceconcordia/spaceconcordia.github.io#16]
On the robotics page:
- Clicking on the altinak image on the robotics page doesn't actually take you to our altinak page
- The altinak page is completely identical to scaar page. The _only_ difference, is the `title` tag.

**Comment by PeterGhimself** [comment]
How do you think we should deal with this issue, is it even an issue? @alienorlgrst

**Comment by alienorlgrst** [comment]
Mmh, Altinak never really existed, we should just remove the link

**Comment by n0pan** [comment]
Yeah I think the altinak page was added towards the end, so no one ever told me what to put on that page

**Comment by PeterGhimself** [comment]
Ok, so maybe we should then also remove the `altiank.html` page from the project?
Perhaps even remove also any reference to it on the page as well? (like the dead link image)
